clean-code/README.md

```diff
@@ -0,0 +1,26 @@
+## Clean code example for Fiber and PostgreSQL
+
+This is an example of a RESTful API built using the Fiber framework (https://gofiber.io/) and PostgreSQL as the database.
```

Comment:
please describe more about the part of the clean code here

Reply:
Added

Reply:
can you please provide further information on cleanup

Reply:
Sorry, I didn't understand what do you mean.